### DISCUSSION TITLE
Mysql database tables not created

### DISCUSSION THREAD

**Discussion by ahmedrhuma** [feathersjs/feathers#2967] [Q&A]
I created application using CLI all good, but the tables are not created, I get the error from mysql that insert into table xxx is not exists.

created new service using CLI also the table is not created, i used migration:make then I deleted the produced migration file. After that all the migration:make files are empty in both up and down functions.

it is so sad to create the tables manually actually.

**Comment by daffl** [comment]
When generating a new application or adding a migration the migrations have to be run via

```
npm run migrate
```

This will create the tables (if they don't exist yet) and run all other migrations. Also see https://feathersjs.com/guides/cli/knexfile.html#migrations

**Reply by ilomon10** [reply]
I cant migrate the file it just show error, i just open issues #3225 
I not familiar with this new version.

Back then at previous version, i just create model and it will automaticly create the schema for database and migrate automaticly
